### PR TITLE
Update docs and add default value to variables

### PR DESCRIPTION
**Pull request by soerenmartius**
Depends on #23 

- feat: add default values for enable_intranode_visibility and private_ipv6_google_access
- docs: update readme and add missing variables

### CODE REVIEW

File: README.tfdoc.hcl [hunk 1]
```diff
@@ -307,57 +533,68 @@ section {
...
-          Cluster autoscaling configuration. For more info see 
-          [documentation](https://cloud.google.com/kubernetes-engine/docs/reference/rest/v1beta1/projects.locations.clusters#clusterautoscaling)
+          Cluster autoscaling configuration. For details please see
+          https://cloud.google.com/kubernetes-engine/docs/reference/rest/v1beta1/projects.locations.clusters#clusterautoscaling
         END
 
-        attribute "enable" {
-          required    = true
+        attribute "enabled" {
           type        = bool
```
Comment (kassianh) [inline README.tfdoc.hcl:542]:
```suggestion
          type        = bool
          default    = false
```

File: README.tfdoc.hcl [hunk 2]
```diff
@@ -422,16 +673,43 @@ section {
...
+      variable "enable_private_nodes" {
+        type        = bool
+        default     = true
         description = <<-END
           Whether nodes have internal IP addresses only.
         END
       }
 
+      variable "private_ipv6_google_access" {
+        type    = bool
```
Comment (kassianh) [inline README.tfdoc.hcl:702]:
```suggestion
        type    = string
```

File: README.tfdoc.hcl [hunk 3]
```diff
@@ -250,30 +347,151 @@ section {
...
+          type        = bool
+          default     = "PROVIDER_UNSPECIFIED"
+          description = <<-END
+            The selected network policy provider.
+          END
+        }
+
         attribute "provider" {
           required    = true
           type        = string
```
Comment (kassianh) [inline README.tfdoc.hcl:360]:
```suggestion
          type        = string
          default     = "CALICO"
```

File: README.tfdoc.hcl [hunk 4]
```diff
@@ -250,30 +347,151 @@ section {
           Configuration options for the NetworkPolicy feature.
         END
 
+        attribute "enabled" {
+          type        = bool
+          default     = "PROVIDER_UNSPECIFIED"
```
Comment (kassianh) [inline README.tfdoc.hcl:352]:
```suggestion
```

File: README.tfdoc.hcl (hunk 3 above)
Comment (soerenmartius) [reply README.tfdoc.hcl:358]: good hint, resolved in c4f7d53a

File: README.tfdoc.hcl (hunk 2 above)
Comment (soerenmartius) [reply README.tfdoc.hcl]: resolved in a1851f72

File: README.tfdoc.hcl (hunk 1 above)
Comment (soerenmartius) [reply README.tfdoc.hcl:542]: resolved in a1851f72

**Review by kassianh — APPROVED** [review]
lgtm!

**Pull request merged**